### PR TITLE
minikube: init at 0.13.1

### PR DESCRIPTION
**Pull request by ebzzry**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
duplicate of https://github.com/NixOS/nixpkgs/pull/20819, but this one is newer.

File: pkgs/applications/networking/cluster/minikube/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+             then "17r8w4lvj7fhh7qppi9z5i2fpqqry4s61zjr9zmsbybc5flnsw2j"
+             else "0jf0kd1mm35qcf0ydr5yyzfq6qi8ifxchvpjsydb1gm1kikp5g3p";
+in
+stdenv.mkDerivation rec {
+  pname = "minikube";
+  version = "0.13.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://storage.googleapis.com/minikube/releases/v${version}/minikube-${arch}";
```
Comment (Mic92) [inline pkgs/applications/networking/cluster/minikube/default.nix]: oh, the other pull request from @offlinehacker  builds from source, so I would actually prefer it in this case.

Comment (ebzzry) [inline pkgs/applications/networking/cluster/minikube/default.nix]: I’ll do that, then I’ll post updates.

Comment (ebzzry) [inline pkgs/applications/networking/cluster/minikube/default.nix]: I can’t build it on macOS. I get `error: attribute ‘glibc’ missing`.

Comment (offlinehacker) [inline pkgs/applications/networking/cluster/minikube/default.nix]: localkube is most probably a problem, as it needs to be static elf x64 binary.

Comment (offlinehacker) [inline pkgs/applications/networking/cluster/minikube/default.nix]: In that particular case i find your pull request better, as I can't see an easy way how to do such build on Darwin, as it needs to be static linux binary, and nix can't do mixed architecture builds. While vm support that nix has could solve this, nix does not support other virtualization as KVM, so this is also no go.

**Comment by offlinehacker** [comment]
Question, I see you don't do patchelf, is minikube statically built?

**Comment by ebzzry** [comment]
@offlinehacker Yes, it does not require patchelf.

**Comment by offlinehacker** [comment]
OK, I will test it tomorrow and then we can merge your pull request.

**Comment by ebzzry** [comment]
Ok!

**Pull request merged**